### DISCUSSION TITLE
Call vehicle when updating vehicle software to 2023.38.9 version_ When the data interface obtains vehicle driving data, it does not return the longitude and latitude where the vehicle is located

### DISCUSSION THREAD

**Discussion by hardont943** [teslamate-org/teslamate#3548] [General]
![image](https://github.com/timdorr/tesla-api/assets/52837734/b042dbc7-fea2-44af-baf0-8320e0779516)
Have you also encountered such a situation?

**Comment by brianmay** [comment]
Is this because the car doesn't have a good GPS position for some reason? (Just speculating here)

**Reply by hardont943** [reply]
I have already dealt with it，thanks！
https://github.com/timdorr/tesla-api/discussions/772#discussioncomment-7806950
